baggers.js

```diff
@@ -17686,7 +17686,60 @@ var data = {
 		    twitter: "alagoye"
       },
     },
+    {
+      since: '2020-09-04',
+      name: 'Delphine Raymond',
+      bio:
+        "Product Owner & Product Manager depuis plus de 7 ans, je travaille chez Lucca et j'ai créé le meetup Product People Nantes, devenu LPCx Nantes, une communauté de PO/PM pour les amoureux de la création de produits numériques.",
+      picture:
+        'https://secure.gravatar.com/avatar/ddbeaab3b87a434071ddc6bfb5af89a8?size=256',
+      websites: [
+        {
+          name: '@Delphine_Ray',
+          url: 'https://twitter.com/Delphine_Ray',
+        }, 
+        {
+          name: 'LinkedIn',
+          url: 'https://www.linkedin.com/in/delphineraymond/',
+        },       
+        {
+          name: 'Lucca',
```

Comment:
It should be Web. See https://github.com/brownbaglunch/bblfr_data#websites

Reply:
Done